src/content/peerconnection/audio/js/main.js

```diff
@@ -30,10 +30,9 @@ var packetSeries;
 var lastResult;
 
 var sdpConstraints = {
-  'mandatory': {
-    'OfferToReceiveAudio': true,
-    'OfferToReceiveVideo': false
-  }
+  offerToReceiveAudio: 1,
```

Comment:
hah. good point about calling this offerOptions, less ambiguous than 'sdpConstraints'